druzhba/postgres.py

```diff
@@ -281,6 +278,20 @@ def query(self, sql):
                 for dict_row in cursor:
                     yield dict_row
 
+    def _get_column_size(self, column_type, internal_size, precision, scale):
```

Comment:
In wonder if it's worth an additional check here, something like `if column_type in self.type_map: return column_type`? Otherwise, it seems like you can end up with applying precision/scale to a totally different type (and I guess the type being mapped to may also be providing its own precision/scale). This may be better suited as a separate concern, though, rather than part of this changeset.

Reply:
I do agree that the type map's precision and scale should be honored but I thought that would be a bigger and potentially breaking change. If you are okay with it I would like to push that off onto a future issue.